RecoBTag/SecondaryVertex/src/TrackSelector.cc

```diff
@@ -71,16 +69,49 @@ TrackSelector::operator () (const Track &track,
   }
   else  jtaPassed = true;
 
+  return track.pt() >= minPt &&
+    VectorUtil::DeltaR(jet.momentum(),
```

Comment:
@slava77, this is now implemented. Sorry for the delay. There was a problem with AFS on lxplus which made my working area inaccessible.